ik_solver_msgs/Configuration[] solutions
string[] joint_names
bool detailed # compute distances and colliding links
moveit_msgs/PlanningScene planning_scene
---
ik_solver_msgs/CollisionResult[] result
